src/util/chat.hpp

```diff
@@ -72,6 +72,8 @@ namespace
 	    "0 Ban Risk",
 	    "Discord For Cheap Money",
 	    "10-30m",
+	    "Hey Guys! Tired of being poor?",
+	    "GTA CASH",
```

Comment:
Yeah sure

Reply:
Question! Instead of having .gg and .GG why don't we just cast the incoming message tolower() and then compare against the list? I don't know how this will affect the 1989 Tiananmen Square Massacre languages, but you know. It would solve at least capitalization permutations.

Reply:
I can implement this

Reply:
Should I implement a manual check(not in the list, and in the for loop itself for the cyrillic text) if the tolower messes it up? Another list that isn't casted could be made too. Let me know your opinions.